### ISSUE TITLE
How to send mass DMs?

### ISSUE THREAD

**Issue by shakesBeardZ** [CamTosh/instagram-bot-dm#9]
I want to send mass DMs using this tool, should I loop through the line of sending because it does make sense, authenticate once and loop to send.
need feedback here.

**Comment by CamTosh** [comment]
Hi, you can do this
```python
from instadm import InstaDM

if __name__ == '__main__':
	# Auto login
	insta = InstaDM(username='your_username', password='your_password', headless=False)

	users = ['user1','user2']
	
	for user in users:
		insta.sendMessage(user=user, message='Hey !')

```

**Comment by gesielbernardes** [comment]
HI, using the exemple code I receive this message:

ERROR:root:Message: no such element: Unable to locate element: {"method":"css selector","selector":"[name="username"]"}
  (Session info: chrome=85.0.4183.121)

The selenium browser opens but nothing happening. How to debug this?

**Comment by kowshika-n** [comment]
This issue is fixed with PR  #10. Please retest once with the latest code.
